Jenkinsfile

```diff
@@ -574,6 +574,7 @@ def withBeatsEnv(Closure body){
     "TEST_COVERAGE=true",
     "RACE_DETECTOR=true",
     "PYTHON_ENV=${WORKSPACE}/python-env",
+    "TEST_TAGS"="aws,azure,oracle",
```

Comment:
Please, don't add aws and azure yet, we have to add credentials and automate the setup of environments.

Reply:
I'm interested in the CI error with aws/azure in TEST_TAGS 😄

Reply:
I'll wait until it finishes before updating the branch :smile:

Reply:
Running aws tests locally I have found that I broke them :man_facepalming:  Preparing fix...

Reply:
Fix here: https://github.com/elastic/beats/pull/17084

When this is fixed, tests fail like this when `TEST_TAGS` includes `aws` and no credentials are provided:
```
--- FAIL: TestFetch (0.00s)
    cloudwatch_integration_test.go:20: $AWS_ACCESS_KEY_ID not set or set to empty
--- FAIL: TestData (0.00s)
    cloudwatch_integration_test.go:33: $AWS_ACCESS_KEY_ID not set or set to empty
FAIL
FAIL	github.com/elastic/beats/v7/x-pack/metricbeat/module/aws/cloudwatch	0.031s
```